src/bosh-google-cpi/integration/vm_test.go

```diff
@@ -855,4 +855,136 @@ var _ = Describe("VM", func() {
 		Expect(justInstances(ig)).ToNot(ContainElement(ContainSubstring(vmCID)))
 	})
 
+	It("executes the VM lifecycle with a backend service, without scheme", func() {
```

Comment:
Do we still have coverage for a backend service with scheme?

Reply:
Yes, the test starting on line 805 covers that case.

Reply:
753 also provides coverage.